### PR TITLE
ATS-290 | Updates in response to internal HMRC accessibility audit

### PR DESCRIPTION
**Pull request by russellthorn**
- remove span from h1
- move dates to h2
- add classes to style appropriately
- add non breaking spaces to the dates
- Remove aria-hidden="true"

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -18,7 +18,7 @@ generic.welsh=Cymraeg
 generic.acting_for=You are acting for
 generic.name_not_defined=Name Not Defined
 generic.tax_year_heading=Tax year: April 6 {0} to April 5 {1}
-paye.generic.tax_year_heading=6 April {0} to 5 April {1}
+paye.generic.tax_year_heading=6&nbsp;April&nbsp;{0} to 5&nbsp;April&nbsp;{1}
```
Comment (oc-ste) [inline conf/messages]: Don't have styling in messages file, it's better to replace this in code so you do not have to do this in both english and welsh

File: test/views/paye/PayeGovernmentSpendViewSpec.scala [hunk 2]
```diff
@@ -90,7 +90,10 @@ class PayeGovernmentSpendViewSpec extends UnitSpec with OneAppPerSuite with Test
 
       document
         .select("h1")
-        .text shouldBe "How your tax was spent 6 April 2018 to 5 April 2019"
+        .text shouldBe "How your tax was spent"
+      document
+        .select("h2.heading-xlarge")
+        .text shouldBe "6 April 2018 to 5 April 2019"
```
Comment (oc-ste) [inline test/views/paye/PayeGovernmentSpendViewSpec.scala]: Should be testing for the new &nbsp; change

File: test/views/paye/PayeTaxFreeAmountViewSpec.scala [hunk 3]
```diff
@@ -50,7 +50,10 @@ class PayeTaxFreeAmountViewSpec extends UnitSpec with OneAppPerSuite with TestCo
 
       document
         .select("h1")
-        .text shouldBe "Tax-free amount 6 April 2018 to 5 April 2019"
+        .text shouldBe "Tax-free amount"
+      document
+        .select("h2.heading-xlarge")
+        .text shouldBe "6 April 2018 to 5 April 2019"
```
Comment (oc-ste) [inline test/views/paye/PayeTaxFreeAmountViewSpec.scala]: Should be testing for the new   change

Comment (Synnnapse) [reply test/views/paye/PayeTaxFreeAmountViewSpec.scala]: Tested in the new spec 👍

File: test/views/paye/PayeGovernmentSpendViewSpec.scala (hunk 2 above)
Comment (Synnnapse) [reply test/views/paye/PayeGovernmentSpendViewSpec.scala]: Tested in the new spec 👍

File: conf/messages (hunk 1 above)
Comment (Synnnapse) [reply conf/messages]: Agreed 👍 done

File: test/utils/HtmlFormatterSpec.scala [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import uk.gov.hmrc.play.test.UnitSpec
+import utils.TaxsValidator._
+
+class HtmlFormatterSpec extends UnitSpec with Matchers {
+
+  "toHtmlNonBroken" should {
+    "replace spaces with non breaking spaces" in {
+      val stringBeforeReplace = "6 April 2018 to 5 April 2019"
+
+      val expectedString = "6&nbsp;April&nbsp;2018&nbsp;to&nbsp;5&nbsp;April&nbsp;2019"
```
Comment (russellthorn) [inline test/utils/HtmlFormatterSpec.scala]: This pattern does not match the accessibility feedback
`6&nbsp;April&nbsp;2018 to 5&nbsp;April&nbsp;2019`
Non breaking spaces  only between the words in the date, not between every word.

**Pull request merged**